### ISSUE TITLE
mpc precision inconsistency

### ISSUE THREAD

**Issue by vinklein** [aleaxit/gmpy#179]
The precision of .real and .imag attributes looks to be inconsistent with their mpc's precision 
```
>>> from gmpy2 import *
>>> x = mpc('56+78j', precision=70)
>>> x.precision
(70, 70)
>>> x.real.precision
53
>>> x.imag.precision
53
```
Is this intended or is it a bug ?

**Comment by casevh** [comment]
It's a bug. I'll work on a fix tomorrow.

**Comment by vinklein** [comment]
Fixed with #181
